charts/pega/templates/_helpers.tpl

```diff
@@ -476,4 +476,20 @@ servicePort: use-annotation
 {{- else -}}
 {{- add .failureThreshold 1 -}}
 {{- end -}}
+{{- end -}}
+
+{{- define "serviceName" -}}
+{{- if or (.Values.hazelcast.enabled) (.Values.hazelcast.skipRestart) -}}
```

Comment:
why skiprestart is needed?

Reply:
To avoid pega pods restart post migration while decommission the old hazelcast pods